testutil/network/network.go

```diff
@@ -174,7 +174,19 @@ func MinimumAppConfig() depinject.Config {
 	)
 }
 
+// DefaultConfigWithAppConfig returns a network configuration constructed using
+// the provided app config. It sets an infinite gas limit on queries by passing zero
```

Comment:
@technicallyty I already mentioned it here though.. Should I move it to `DefaultConfigWithAppConfig` godoc instead ?